webapp/patches/enketo-inputs-always-relevant.patch

```diff
@@ -0,0 +1,20 @@
+*** unpatched	2022-04-05 11:46:20.970352329 -0500
+--- patched	2022-04-05 11:50:47.743949897 -0500
+***************
+*** 216,222 ****
+  
+          if ( neverEnabled || this.selfRelevant( branchNode ) || forceClearNonRelevant ) {
+              changed = true;
+!             if ( forceClearNonRelevant ) {
+                  this.clear( branchNode, path );
+              }
+  
+--- 216,223 ----
+  
+          if ( neverEnabled || this.selfRelevant( branchNode ) || forceClearNonRelevant ) {
+              changed = true;
+!             // /inputs is ALWAYS relevant #4875
+!             if ( forceClearNonRelevant && !/\/inputs$/.test(path) ) {
```

Comment:
I think this will end up generating the regex every time this code runs, which is much less performant than using a simple `endsWith` (`path.endsWith('/inputs')`): https://jsbench.me/ytl1w18gtr/1